spec/order_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'order'
+
+describe Order do 
+
+  test_menu = Takeaway.new.menu
+  let(:cheeseburger) { Takeaway.new.menu[10] }
+
+  it 'should accumulate selected dishes from takeaway' do 
+    cheeseburger = "cheeseburger"
+    allow(subject).to receive(:selected_dishes).and_return(cheeseburger)
+    expect(subject.selected_dishes).to include cheeseburger
+  end 
+
+  it 'should add up total cost of selected dishes' do
+    subject.add_dish("Cheeseburger")
+    subject.add_dish("Hot dog")
+    expect(subject.total).to eq 17
+  end 
+
+end 
```

Comment:
not sure what the test coverage score was on this but to get it higher you could test that @selected_dishes and @total are empty arrays on initializing?